third_party/move/move-compiler-v2/src/pipeline/control_flow_graph_simplifier.rs

```diff
@@ -469,6 +496,30 @@ impl ControlFlowGraphCodeGenerator {
                         .collect::<BTreeSet<_>>()
             );
         }
+        // check invariant 3
+        for (block, code) in self.code_blocks.iter() {
+            if block == &self.entry_block || block == &self.exit_block {
```

Comment:
Can you use `is_trivial_block()` here instead, instead of the logic duplication?

Reply:
done